README.md

```diff
@@ -10,62 +10,86 @@ Please find project details at [Wiki](https://github.com/Samsung/TizenRT/wiki) e
+To install OS specific Docker engines, see https://docs.docker.com/install/linux/docker-ce/ubuntu/.  
+If you already have a Docker engine, please skip this step.
 
-### Getting the sources
+### 2. Getting TizenRT source code
 
 ```bash
 git clone https://github.com/Samsung/TizenRT.git
 cd TizenRT
 TIZENRT_BASEDIR="$PWD"
 ```
+**Note**: To contribute in this community, fork this project first into own github repository and do git cloning from your repository. 
+> git clone https://github.com/your_github_id/TizenRT.git
+
+
+### 3. Build
 
-### How to Build
+#### 3.1 Configuration (*$TIZENRT_BASEDIR/os/tools*)
 
-Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
 ```bash
 cd os/tools
 ./configure.sh <board>/<configuration_set>
 ```
-The configuration file is named *defconfig*,  
-and resides under the relative path \<board\>/\<configuration_set\> rooted at *build/configs*.  
-To check the different \<board\>/\<configuration_set\> combinations supported, type below:
+
+This command retrieves the specific pre-configured file named defconfig according to
+ \<board\>/\<configuration_set\>.  
+You can see collection of all configuration files at
+ *$TIZENRT_BASEDIR/build/configs*.  
+To check all pre-defined configurations, type as follows:
+
 ```bash
 ./configure.sh --help
 ```
 
-After configuring above, configuration can be modified through *make menuconfig* from *$TIZENRT_BASEDIR/os*.
+#### 3.2 Additional Configuration (*$TIZENRT_BASEDIR/os*)
+
+After the configuration is done, further modification can be done with menuconfig from
+ *$TIZENRT_BASEDIR/os*.  
+This command requires ```sudo``` for root permission.  
+To run Docker without ```sudo```, refer to
+ https://docs.docker.com/install/linux/linux-postinstall/.
+
 ```bash
 cd ..
-make menuconfig
+./dbuild.sh menuconfig
+```
+
+#### 3.3 Compile (*$TIZENRT_BASEDIR/os*)
+
+```bash
+./dbuild.sh
 ```
 
-Refer [kconfig-frontend installation](docs/HowtoInstallKconfigFrontend.md) to use *menuconfig*.
+Built binaries are located in *$TIZENRT_BASEDIR/build/output/bin*.
+
+#### 3.4 Clean (*$TIZENRT_BASEDIR/os*)
 
-Finally, initiate build by make from *$TIZENRT_BASEDIR/os*.
 ```bash
-make
+./dbuild.sh clean
 ```
+This command removes built files including objects, libraries, .depend, Make.dep, etc.  
+To modify configuration with menuconfig, this command is required.  
 
-Built binaries are in *$TIZENRT_BASEDIR/build/output/bin*.
+Before changing configuration with ```./configure.sh``` command, deletion of previous configuration files must be done using the follwoing command.
 
-See [Clean commands](docs/HowtoClean.md) to clean built files.  
-See [Trouble Shooting](docs/TroubleShooting.md) to resolve any issue on TizenRT usages.
+```bash
+./dbuild.sh distclean
+```
+This command removes configured files including .config and Make.defs.
 
 ## Supported Board / Emulator
 Here are supported boards and emulator list.  
```

Comment:
I suggest the following paragraph:
TizenRT supports multiple boards as well as QEMU.
The linked page for each board includs board-specific environments, programming method, and board information.